tests/controls/dataviewctrltest.cpp

```diff
@@ -296,14 +346,66 @@ class DataViewCtrlWithCustomModelTestCase
     ~DataViewCtrlWithCustomModelTestCase();
 
 protected:
+    enum wxItemExistance
```

Comment:
There is a typo here
```suggestion
    enum wxItemExistence
```
(and also in `existance` variable/parameter name below, it should be `existence`).